docs: add FAQ entry about subpixel anti-aliased text

heronils opened the pull request:
#### docs: Add FAQ entry about subpixel anti-aliased text

Added an entry to docs/faq.md, which explains the reason for blurry looking fonts: A transparent font layer. It further explains how to fix it: By setting `backgroundColor` to a non-transparent color in the constructor of BrowserWindow. This will enable subpixel anti-aliased text and the font will stop looking blurry on LCD screens.

Resolves: #10955

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

crypto480 commented:
Quit doing dope

malept commented on docs/tutorial/blurry-font.md:
```diff
@@ -0,0 +1,16 @@
+If fonts look blurry for you, set the window background to a non transparent color.
+Do this in the constructor for [BrowserWindow][browser-window]:
+
+    new BrowserWindow({width: 1000, height: 750, backgroundColor: '#fff'});
```
This should be in a fenced Markdown code block (for JavaScript).

malept commented on docs/tutorial/blurry-font.md:
```diff
@@ -0,0 +1,16 @@
+
+Screenshot of the result:
+
+![subpixel rendering example](https://i.imgur.com/CjNQaeR.gif)
```
We probably don't want to use linked images from imgur.

heronils commented:
Hello @malept Sure, let this be a FAQ entry, with or without image. Please rewrite it / remove what you dont like.

For the records, this is how the official API docs look with and without the 'Hack':

![electron api demos](https://user-images.githubusercontent.com/6892179/52511443-4a51ae80-2c00-11e9-893e-74bb691c921c.gif)

malept commented:
If you'd like to move the docs to a FAQ entry, the document is at `docs/faq.md`.

heronils commented:
Ok, im just trying to do this.

heronils commented:
OK, that should be good now.

heronils commented:
Sorry, cant make release bot happy, it seems.

jkleinsc commented:
@heronils The release-notes bot is unhappy because you deleted the issue template we use, specifically:
#### Description of Change
<!--
Thank you for your Pull Request. Please provide a description above and review
the requirements below.

Contributors guide: https://github.com/electron/electron/blob/master/CONTRIBUTING.md
-->

#### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).


#### Release Notes

Notes: <!-- Please add a one-line description for app developers to read in the release notes, or `no-notes` if no notes relevant to app developers. Examples and help on special cases: https://github.com/electron/clerk/blob/master/README.md#examples -->

heronils commented:
Well, i tried :-)

malept commented:
> i inserted this text, still no success, what else am i missing?

Edit https://github.com/electron/electron/pull/16459#issue-246023306 with the [pull request template](https://github.com/electron/electron/blob/master/.github/PULL_REQUEST_TEMPLATE.md), filling it out as appropriate.

heronils commented:
@malept thanks for helping me out again, did this. Now build-linux and build-mac-fork-prs still fails but this has nothing to do with my changes, as i did not change any code files. I just added an image to docs/images and i changed docs/faq.md.

miniak commented:
@heronils release notes are only meant for user visible changes (APIs, bug fixes, etc.), updated docs should not be mentioned.

jkleinsc commented:
@heronils I closed this PR in an attempt to get our lint process to verify the changes by closing and reopening the PR, but it appears that the repository is no longer available.  Feel free to reopen this PR if you still have the repository.

jkleinsc commented:
Reopened this PR in #18101.
